.github/files/create-e2e-projects-matrix.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eo pipefail
 
-PROJECTS=('{"project":"Jetpack connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack pre-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/pre-connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack post-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/post-connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack sync","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/sync","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack blocks","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/blocks","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack update","path":"projects/plugins/jetpack/tests/e2e","testArgs":["plugin-update","--retries=2"],"slackArgs":[]}' '{"project":"Boost","path":"projects/plugins/boost/tests/e2e","testArgs":[],"slackArgs":[]}' '{"project":"Search","path":"projects/plugins/search/tests/e2e","testArgs":[],"slackArgs":[]}' '{"project":"VideoPress","path":"projects/plugins/videopress/tests/e2e","testArgs":[],"slackArgs":[]}')
+PROJECTS=('{"project":"Jetpack connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack pre-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/pre-connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack post-connection","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/post-connection","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack sync","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/sync","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack blocks","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/blocks","--retries=2"],"slackArgs":[]}' '{"project":"Jetpack update","path":"projects/plugins/jetpack/tests/e2e","testArgs":["plugin-update","--retries=2"],"slackArgs":[]}' '{"project":"Boost","path":"projects/plugins/boost/tests/e2e","testArgs":[],"slackArgs":[]}' '{"project":"Search","path":"projects/plugins/search/tests/e2e","testArgs":[],"slackArgs":[]}' '{"project":"VideoPress","path":"projects/plugins/videopress/tests/e2e","testArgs":[],"slackArgs":[]}' '{"project":"Jetpack waf","path":"projects/plugins/jetpack/tests/e2e","testArgs":["specs/waf","--retries=2"],"slackArgs":[]}')
```

Comment:
Do we need to add a separate CI action here?

Reply:
Well, there weren't any WAF e2e tests before, I guess we could include them in some of the other directories, but they all are pretty much different topics. Where else would we best group it under?

Reply:
Our CI matrix is organized with efficiency in mind. `pos-connection` probably would be the best place for them.

The problem with additional action is that it will increase the total number of actions being run for a PR, which is something I'd like to avoid if possible. Also, the test itself should be fast enough, not to slow down the `post-connection` action significantly.

Reply:
From the looks of it, the tests in "post-connection" provision a Free plan. Does that make any difference to the reasoning behind putting it in the same group? Or should we put it in there regardless so that we still save the extra CI job?

Reply:
> From the looks of it, the tests in "post-connection" provision a Free plan.

That's true, but it should be ok since these tests are actually mocking plans and running sequentially.

Reply:
Moved it into the "post-connection" group :)